src/components/navbar/SocialLinks.tsx

```diff
@@ -0,0 +1,40 @@
+import { SocialLink } from 'components/ui/Text'
+import { displayFrom } from 'helpers/visibilityClassnames'
+import Discord from 'icons/Discord'
+import Twitter from 'icons/Twitter'
+import classnames, {
+  alignItems,
+  backgroundColor,
+  borderWidth,
+  display,
+  height,
+  space,
+  width,
+} from 'classnames/tailwind'
+
+const socialContainer = classnames(
+  display('inline-flex'),
+  alignItems('items-center'),
+  space('space-x-4')
+)
+const delimiterContainer = classnames(
```

Comment:
Just like in `LoadingWallet`, probably worth extracting or making one component from two